### PR TITLE
(Fix) display correct min cap in contribution page

### PR DESCRIPTION
**Pull request by vbaranov**
Closes https://github.com/poanetwork/token-wizard/issues/977
Closes https://github.com/poanetwork/token-wizard/issues/978

display correct min cap in contribution page

### CODE REVIEW

**Comment by dennis00010011b** [comment]
@vbaranov 
This issue still reproducible for Dutch
`0xbd5c86c9fbaa47a598cf55ad1cee7a1d8f8450857cb10e0f122503b8c0e65243 `
 Ropsten
screen shot 2018-06-26 at 13 26 14

File: src/components/crowdsale/utils.js [hunk 1]
```diff
@@ -358,13 +358,16 @@ const getRate = async (addr, execID, methods) => {
 
 const calculateMinContribution = async (method, decimals, naturalMinCap, isWhitelisted) => {
   //todo: update for Proxy
-  const { minimum_contribution, minimum_purchase_amt, max_tokens_remaining, max_purchase_remaining } = await method.call()
-  const minimumContribution = toBigNumber(minimum_contribution).times(`1e-${decimals}`)
+  const { tier_min, minimum_contribution, minimum_purchase_amt, max_tokens_remaining, max_purchase_remaining } = await method.call()
+  const minimumContributionDutch = toBigNumber(minimum_contribution).times(`1e-${decimals}`)
+  const minimumContributionMintedCapped = toBigNumber(tier_min).times(`1e-${decimals}`)
+  const minimumContribution = minimumContributionMintedCapped || minimumContributionDutch
```
Comment (fernandomg) [inline src/components/crowdsale/utils.js]: Still is returning `naturalMinCap` as `toBigNumber` is returning a bigNumber object with value `0` when the parameter passed is not a number or a string representation of a number.

A solution would be to replace lines `362..364` with:
```
const minimumContribution = toBigNumber(tier_min || minimum_contribution).times(`1e-${decimals}`)
```

But I think that after this, in a different PR, we should replace `toBigNumber` signature and return `undefined` when an invalid value is passed.

What you think @vbaranov ?

Comment (vbaranov) [reply src/components/crowdsale/utils.js]: Yes, agree. It would be nice to mark invalid values as `undefined`

**Comment by vbaranov** [comment]
@fernandomg @dennis00010011b fixed, thank for your suggestions. Also, I've added a fix for https://github.com/poanetwork/token-wizard/issues/978, as it is related to the same page component.

**Pull request merged**